docs/pages/web/utilities/globaleventshandlerprovider.js

```diff
@@ -0,0 +1,53 @@
+// @flow strict
+import { type Node } from 'react';
+import docgen, { type DocGen } from '../../../docs-components/docgen.js';
+import GeneratedPropTable from '../../../docs-components/GeneratedPropTable.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import SandpackExample from '../../../docs-components/SandpackExample.js';
+import SheetMobileHandlers from '../../../examples/globaleventshandlerprovider/sheetMobileHandlers.js';
+
+export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader
+        name={generatedDocGen?.displayName}
+        description={generatedDocGen?.description}
+        type="utility"
+      />
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection title="GlobalEventsHandlerProvider in SheetMobile">
+          <MainSection.Card
+            cardSize="lg"
+            description={`GlobalEventsHandlerProvider has one prop for each component subscribing to the provider.
```

Comment:
This copy needs to be updated. It still references the previous Provider name, and is a bit vague about if this affects all usages of a single component or if it needs to be implemented at the component callsite. Consider something like
"GlobalEventsHandlerProvider is a React Context Provider that is intended to be implemented at the top level of your app or feature. Each prop corresponds to a specific Gestalt component, and allows you to add logic that will run for every instance of that component in the component tree.

For SheetMobile, we can add `onOpen` and `onClose` handlers. `onOpen` runs on component mount, when the SheetMobile first appears in the UI. `onClose` runs just before the component unmounts and is removed from the UI."

Reply:
DONE